### ISSUE TITLE
llsubmit error with ASAGI 

### ISSUE THREAD

**Issue by git-taufiq** [SeisSol/SeisSol#46]
Hi all,

I am trying to use ASAGI to input the stress tensor and dynamic parameter NetCDF in fault.yaml
I got this error messages while llsubmit job.
'''
Abort(1) on node 29 (rank 29 in comm 1140850688): Fatal error in MPI_Allreduce: Message truncated, error stack:
MPI_Allreduce(912).......: MPI_Allreduce(sbuf=0x7ffd066738b8, rbuf=0x7ffd06677710, count=1, MPI_DOUBLE_PRECISION, MPI_MIN, MPI_COMM_WORLD) failed
MPIR_Allreduce_impl(769).:
MPIR_Allreduce_intra(419):
MPIC_Sendrecv(467).......:
MPIDI_Buffer_copy(73)....: Message truncated; 260 bytes received but buffer size is 8
'''
Do you have any idea to fix this problem?

**Comment by Thomas-Ulrich** [comment]
Hi,
It looks like a bug in asagi/easi. It seems that the case in which all partitions dont have a fault boundary condition has not been properly handled.
In particular, sebastian designed a specific mpi communicator containing only the ranks with fault boundary that could be used instead the communicator containing all ranks. 
Thomas.

**Comment by git-taufiq** [comment]
I just found out the source of the problem. It is because of the mesh that I generated by Gmsh (convert .neu to .h5 & .xdmf using PUMGen without Simmetrix). 

SeisSol+ASAGI worked well for passing material properties (Rho, Mu, Lambda) NetCDF file on the material domain case, however SeisSol+ASAGI failed passing stresses, dynamic parameters NetCDF file on the fault domain case.

I found out this by testing SeisSol with the mesh created using PUMGen with Simmetrix. Surprisingly, SeisSol and ASAGI worked well passing those parameters to the fault domain.

So in the case of using ASAGI to passing dynamic parameters to the fault domain, I need to use the mesh that generated using PUMGen with Simmetrix analysis .smd file, ie.:
…
pumgen -s simmodsuite -l SimModelerLib.lic --mesh "Mesh case 1" --analysis "Analysis case 1" test.smd test
…

**Comment by git-taufiq** [comment]
It solved by adding this line in your job file:
export SEISSOL_ASAGI_MPI_MODE=OFF

**Comment by Thomas-Ulrich** [comment]
This solves your problem but the bug in ASAGI is not fixed. Therefore issue reopened.

**Comment by davschneller** [comment]
Is this issue still reproducible?

(if it's a communicator thing—that might even be pretty straight-forward to introduce)

**Comment by Thomas-Ulrich** [comment]
I don't think so. At least I never had such an issue since a long time and none of my simulations set SEISSOL_ASAGI_MPI_MODE. 
Let's close it.
